### PR TITLE
Fix foot down calculation

### PR DESCRIPTION
**Pull request by Bidski**

### CODE REVIEW

File: shared/message/input/Sensors.proto [hunk 1]
```diff
@@ -51,8 +52,9 @@ message Sensors {
         uint32 colour = 2;
     }
     message Foot {
-        bool down = 1;
-        mat4 Hwf  = 2;
+        bool            down = 1;
+        mat4            Hwf  = 2;
+        motion.BodySide side = 3;
```
Comment (TrentHouliston) [inline shared/message/input/Sensors.proto:57]: Not 100% sure this is needed here as the index already implies it...

Comment (TrentHouliston) [reply shared/message/input/Sensors.proto:57]: Actually, 100% remove this, people will think changing side == changing the meaning. But others will use it as an index and we have a problem.

**Comment by Bidski** [comment]
You saw the nbs file @TrentHouliston , you tell me of it worked

**Comment by TrentHouliston** [comment]
Probably kinda maybe?

**Comment by Bidski** [comment]
Good enough for me

**Pull request merged**